README.rdoc

```diff
@@ -88,6 +88,16 @@ Then for your Enterprise projects, you have to setup GitReflow for each one:
   cd replace_with_your_enterprise_project_path
   git reflow setup --enterprise --local
 
+=== Refreshing your current branch based on your base branch
+
+  git reflow refresh
+
+This command updates your feature_branch and base_branch according to the remote_location and then merges your feature_branch with the base_branch. This is just a handy command to keep your branches up to date at any point in time if someone else has committed to the base_branch or the remote.
+
+  git reflow refresh <remote_location> <base_branch_name>
```

Comment:
I find the [GLI wiki](https://github.com/davetron5000/gli/wiki/Declaring-flags-and-switches) has better docs at times :wink:.  So with the new flags you get `git reflow refresh -r upstream -b branch-name` or `git reflow refresh --remote=upstream --branch=branch-name`